### DISCUSSION TITLE
Open BoxSizeRequest in Styles

### DISCUSSION THREAD

**Discussion by pouchbunny** [enisn/UraniumUI#810] [Ideas]
Is it possible to open the box size in styles, just to keep from having to repeat so much of the same data on multiple items?

![image](https://github.com/user-attachments/assets/6e88f0ca-f06b-4f81-8d56-c014c310b96b)

**Comment by enisn** [comment]
It seems it's not a bindable property. I need to add a bindable property for it, thanks for reporting

**Reply by pouchbunny** [reply]
No problem, thanks for all the hard work with this  =)
